src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1481,7 +1481,7 @@ namespace winrt::TerminalApp::implementation
             }
             if (!profileFound)
             {
-                std::tie(realGuid, controlSettings) = _settings->BuildSettings(newTerminalArgs);
+                std::tie(realGuid, controlSettings) = _settings.as<CascadiaSettings>()->BuildSettings(newTerminalArgs);
```

Comment:
Changed `_settings` to be the projected type since we'll need that for TerminalSettingsModel.

For now, I'm getting the impl type when I need it for `BuildSettings`. But #7537 moves this logic to `TerminalSettings` so it'll all go away soon.